From 3c029d1d6858aaa232e9706f847c390addf0fd96 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Thu, 12 Sep 2019 20:38:08 +0200
Subject: [PATCH 31/58] Check for samsung light service everytime

This is needed because it is possible sec light service isn't ready at
that time.
TODO: check that the services exists at all, so that this is done only
on Samsung devices

Change-Id: I30f049f3b06f83c455301b589b3558ff384ec300
---
 services/core/jni/com_android_server_lights_LightsService.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/jni/com_android_server_lights_LightsService.cpp b/services/core/jni/com_android_server_lights_LightsService.cpp
index 2b50d04233e..a21c6e2c8ef 100644
--- a/services/core/jni/com_android_server_lights_LightsService.cpp
+++ b/services/core/jni/com_android_server_lights_LightsService.cpp
@@ -161,7 +161,7 @@ static void setLight_native(
 
     if(!sSecTried) {
         sSecHal = ISecLight::getService();
-        sSecTried = true;
+        //sSecTried = true;
     }
 
     if(sSecHal != nullptr) {
-- 
2.25.1

